db/db.js

```diff
@@ -379,6 +379,24 @@ async function getLocationData(locationid) {
 
 }
 
+// Retrieves the locationid corresponding to a particle device coreID
+
+async function getLocationFromCoreID(coreID){
+    try{
+        const results = await pool.query("SELECT (locationid) FROM locations WHERE door_coreid = $1 OR radar_coreid = $1", [coreID])
```

Comment:
OK, if you'd like to leave this one as getting the `Location`, then it would be better if this were a `SELECT *` instead of just a `SELECT locationid` and then map the resulting rows in `Location` objects from `Location.js`.

In fact, this reminds me that `Location.js` should be updated to include the two new columns and then you should check on all the DB functions that use it to make sure that they all take those new columns into consideration.